libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -401,12 +402,29 @@ public MapboxMap getMapboxMap() {
     return map;
   }
 
+  public void setInstructionViewCallback() {
+    if (instructionView != null) {
+      instructionView.setInstructionListListener(new InstructionListListener() {
+        @Override
+        public void onInstructionListShown(boolean shown) {
```

Comment:
It may make more sense to update the naming here to something like `onInstructionListVisibilityChanged(boolean isShowing)` because the listener is fired for both the list showing and hiding and not just showing as the current naming may imply